programs/tests/c-tests/time/time.c

```diff
@@ -0,0 +1,21 @@
+#include <stdint.h>
```

Comment:
would be nice to test the edge cases in `rdtime64`, but not sure how to accomplish since that would require simulating on the order of 2^32 cycles

Reply:
We could allow writes to the counter register. Might be useful for clock synchronization. But might be very dangerous also